homeworks/Aleksandr.Turuta_turuta7/homework_7/homework_7.js

```diff
@@ -0,0 +1,18 @@
+function makeFiboArr(rand) {
+    const array = [0, 1];
+    const randNum = rand === undefined ? 50 : rand;
+    console.log(randNum);
+    for (let i = 0; i < randNum; i++) {
```

Comment:
one issue remaining here... your loop will have `randNum` iterations always, there may be a number between 50 and 100, but your Fibo array will be done already after 10 or so iterations, that said you are using resources for actions that don't have any sense, to solve this you need to `break` out of your loop when `fib < randNum` condition is already not met 

also... why not start looping from 2? you first two elements of array is already set...